technical.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Technical Patterns
-description: "Yadayada."
+description: "Patterns that focus on how to manage and operate Cloud Native services whilst ensuring their reliability and security."
```

Comment:
I have tried to add a description but its a work in progress (and def better than "yadayada").

Would def appreciate input from others to try and get it to a sentence that encompasses what these patterns are about.

Reply:
Looks pretty close to me. Is this an improvement?

```suggestion
description: "Patterns that focus on the technical and architectural aspects of how to build and operate Cloud Native services whilst ensuring their reliability and security."
```

Reply:
Thanks. That is better now.